msal-python-conceptual/advanced/wam.md

```diff
@@ -7,19 +7,16 @@ manager: CelesteDG
 ms.service: msal
 ms.subservice: msal-python
 ms.topic: conceptual
-ms.date: 02/07/2024
+ms.date: 04/24/2025
 ms.author: dmwendia
 ms.reviewer: shermanouko, rayluo
 ---
 
 # Using MSAL Python with Web Account Manager
 
-If you are building a Windows application, you might consider simplifying how users authenticate with the help of an _authentication broker_ - the [Web Account Manager](/windows/uwp/security/web-account-manager) (WAM).
+If you are building a Windows application, you might consider simplifying how users authenticate with the help of an *authentication broker*. [Web Account Manager](/windows/uwp/security/web-account-manager) (WAM) is an authentication broker that works with MSAL Python. WAM is only available on Windows 10 and above, as well as Windows Server 2019 and above.
 
->[!NOTE]
```

Comment:
Why is this note being removed? For enterprise scenario, it's criticial to know where broker can and can't be used (e.g., Windows Server 2016 customers might have issues).